SafeSurroundings/SafeSurroundings.Data/Services/InMemoryMeetUpTable.cs

```diff
@@ -28,9 +28,21 @@ public void Delete(int id)
             meetUps.Remove(meetup);
         }
       
-        public void Update(int id)
+        public MeetUp Update(MeetUp meetupToUpdate)
         {
-            throw new NotImplementedException();
+            try
+            {
+                MeetUp meetUpFromUpdate = meetUps.Where(m => m.ID == meetupToUpdate.ID).FirstOrDefault();
+                meetUpFromUpdate.MeetTime = meetupToUpdate.MeetTime;
+                meetUpFromUpdate.PlaceName = meetupToUpdate.PlaceName;
+                meetUpFromUpdate.PlaceID = meetupToUpdate.PlaceID;
+                return meetUpFromUpdate;
+            }
+            catch
+            {
+                return null;
+            }
+         
```

Comment:
Making this a simpler datatype on the parameter. The interface implements an integer and switches to a database, it will need to pass the ID anyway.